src/mat/4C_mat_micromaterial.hpp

```diff
@@ -157,6 +167,12 @@ namespace Mat
     /// Calculate stresses and strains on the micro-scale
     void prepare_output();
 
+    /// Calculate stresses and strains on the micro-scale
+    void prepare_runtime_output() const;
+
+    /// Write output on micro-scale
```

Comment:
Do you somewhere document the strategy behind the micro scale output and the options, e.g., what kind of quantities can be output, that the stiffness matrix is output as field data, ... . That would greatly benefit the future use and extendability of the micro scale output.